doc/howto/README.md

```diff
@@ -0,0 +1,31 @@
+= FreeRADIUS Howto
+
+List with some usual howtos for FreeRADIUS.
+
+## Topics
+
+* [Ascend](ascend.md)
+* [Bay](bay.md)
+* [Cisco](cisco.md)
+* modules
+	* [RADIUS-LDAP-eDirectory](modules/RADIUS-LDAP-eDirectory.md)
```

Comment:
Don't usually use tabs as it's a PITA to edit in any web-based editor.  Also... why are we using markdown for half the documentation and asciidoc for the other half?

Reply:
1. The reason for the tabs, It is because of the `Mixed List` syntax. Take a look in `Mixed List` 
 at https://en.support.wordpress.com/markdown-quick-reference/

2. I am just reusing the current markdown files that are converted by pandoc.

Reply:
A good chunk of the markdown is old documentation.  So that can be gradually fixed.

In general, stand-alone pages are simpler in markdown.  Multi-page sites are better in Asciidoc